requirements.txt

```diff
@@ -1,4 +1,5 @@
 cloudpickle==2.0.0
+dask[distributed]==2022.1.0
```

Comment:
Hey @Emmanuel289 , quick comment, instead of this, please make our dask executer plugin a requirement, which intern has dask and other things.